Public/Src/Cache/ContentStore/Distributed/NuCache/CheckpointManager.cs

```diff
@@ -112,10 +122,10 @@ public Task<BoolResult> CreateCheckpointAsync(OperationContext context, EventSeq
                     }
                 },
                 extraStartMessage: $"SequencePoint=[{sequencePoint}]",
-                extraEndMessage: _ => $"SequencePoint=[{sequencePoint}]");
+                extraEndMessage: result => $"SequencePoint=[{sequencePoint}] Id=[{checkpointId}] Size=[{checkpointSize}]");

```

Comment:
Another minor comment. Does it make sense to print the size in megabytes instead? bytes is too low level imo.